x-pack/plugins/ingest_pipelines/public/application/components/pipeline_editor/__jest__/processors/csv.test.tsx

```diff
@@ -0,0 +1,172 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { act } from 'react-dom/test-utils';
+import { setup, SetupResult, getProcessorValue } from './processor.helpers';
+
+// Default parameter values automatically added to the CSV processor when saved
+const defaultCSVParameters = {
+  description: undefined,
+  if: undefined,
+  ignore_missing: undefined,
+  ignore_failure: undefined,
+  empty_value: undefined,
+  quote: undefined,
+  separator: undefined,
+  tag: undefined,
+  trim: false,
```

Comment:
I see `trim` is set to a boolean (`false`) as the default, but the other toggles - `ignore_missing` and `ignore_value` - are `undefined`. Is that correct behavior?

Reply:
Upon further investigation, I see the `trim` field does not have a `serializer` defined like the other fields. I'm guessing this is an oversight, as I don't see any reason for it to behave differently. I can open up an issue to follow up on this. I think it also might be a small enough change to to contain within this PR (while we're addressing issues with the CSV processor). Let me know what you think!

Reply:
Ohh thats a very nice catch! I've added another small commit to add the missing serializer :)

Reply:
Thanks!